src/main/java/hu/flowacademy/zetaabsencemanager/service/GroupService.java

```diff
@@ -23,4 +26,14 @@
     public Optional<Group> findOne(Long id){
         return groupRepository.findById(id);
     }
+
+    public Group create(Group group) {
+        if (group.getId() != null || group.getName().equals("") || group.getName() == null ||
```

Comment:
`StringUtils.isEmpty(group.getName())` legyen, ez checkeli a null-t is.
van `CollectionUtils` ami ugyan ezt tudja kollekciókra, használd azt is.